glean-core/python/glean/metrics/ping.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from ..glean import Glean
+from .. import _ffi
+
+
+class PingType:
```

Comment:
Mh, should we update the [docs here](https://mozilla.github.io/glean/book/user/pings/custom.html) too? Can be a follow-up bug.